diff --git a/Formula/emacs-mac.rb b/Formula/emacs-mac.rb
index bc51391..737dd8a 100644
--- a/Formula/emacs-mac.rb
+++ b/Formula/emacs-mac.rb
@@ -85,6 +85,11 @@ class EmacsMac < Formula
     "https://raw.githubusercontent.com/emacsfodder/emacs-icons-project/master/#{icon}.icns"
   end
 
+  patch do
+    url "https://gist.githubusercontent.com/choppsv1/e8198df70cb1778b04d6/raw/4bb449f36b621ae5122b673bf40cfad0b3bb103e/emacs-24.5-24bit.diff"
+    sha256 "fbe9ce2014535f79d878d65062906f8a869bf49fe51421672f10a406e1d44b12"
+  end
+
   def install
     args = [
       "--prefix=#{prefix}",
