diff --git a/Formula/emacs-mac.rb b/Formula/emacs-mac.rb
index 9081542..9ba919c 100644
--- a/Formula/emacs-mac.rb
+++ b/Formula/emacs-mac.rb
@@ -73,6 +73,11 @@ class EmacsMac < Formula
     sha256 "b3db8b7cfa4bc5bce24bc4dc1ede3b752c7186c7b54c09994eab5ec4eaa48900"
   end
 
+  patch do
+    url "https://gist.githubusercontent.com/choppsv1/e8198df70cb1778b04d6/raw/4bb449f36b621ae5122b673bf40cfad0b3bb103e/emacs-24.5-24bit.diff"
+    sha256 "fbe9ce2014535f79d878d65062906f8a869bf49fe51421672f10a406e1d44b12"
+  end
+
   def install
     args = [
       "--enable-locallisppath=#{HOMEBREW_PREFIX}/share/emacs/site-lisp",
