diff --git a/Library/Formula/git.rb b/Library/Formula/git.rb
index 33a8183..4fbdc37 100644
--- a/Library/Formula/git.rb
+++ b/Library/Formula/git.rb
@@ -22,6 +22,12 @@ class Git < Formula
     sha256 "515bee47dbb1f1c345eae394e14f82009865529aaf0c201301defee32512a7c6"
   end
 
+  patch do
+    url "https://gist.githubusercontent.com/bsuh/c358ccd9da982a30a966/raw/e9c9925045f3b766390c1828c9b60ac0fb5967dd/gistfile1.diff"
+    sha256 "3d9817b33de714c9f97b5ac579de66320be6bb4d697c81b2aa0484ecf061c1e0"
+  end
+
+
   option "with-blk-sha1", "Compile with the block-optimized SHA1 implementation"
   option "without-completions", "Disable bash/zsh completions from 'contrib' directory"
   option "with-brewed-openssl", "Build with Homebrew OpenSSL instead of the system version"
