diff --git a/Formula/git.rb b/Formula/git.rb
index 956aa0f..4d88061 100644
--- a/Formula/git.rb
+++ b/Formula/git.rb
@@ -40,6 +40,11 @@ class Git < Formula
     sha256 "83b0a317f6039ad95ef6af6a182cf659c2d053eed5d8f70b06710eb787f8aa6f"
   end
 
+  patch do
+    url "https://gist.githubusercontent.com/bsuh/c358ccd9da982a30a966/raw/e9c9925045f3b766390c1828c9b60ac0fb5967dd/gistfile1.diff"
+    sha256 "3d9817b33de714c9f97b5ac579de66320be6bb4d697c81b2aa0484ecf061c1e0"
+  end
+
   def install
     # If these things are installed, tell Git build system not to use them
     ENV["NO_FINK"] = "1"
