diff --git a/Library/Formula/git.rb b/Library/Formula/git.rb
index 528e7b5..23f29b8 100644
--- a/Library/Formula/git.rb
+++ b/Library/Formula/git.rb
@@ -22,6 +22,12 @@ class Git < Formula
     sha256 "a9d30a744eb07392ba3ffe389cca7e57a2bdd718c9ffccdf2d525eeafac548a9"
   end
 
+  patch do
+    url "https://gist.githubusercontent.com/bsuh/c358ccd9da982a30a966/raw/3ee52ae5b5712b3c58a6d79af7800f59f65bbd96/gistfile1.diff"
+    sha1 "cd7b5e88345ddc0e7cf83147eabcb756c4a73fe0"
+  end
+
+
   option "with-blk-sha1", "Compile with the block-optimized SHA1 implementation"
   option "without-completions", "Disable bash/zsh completions from 'contrib' directory"
   option "with-brewed-openssl", "Build with Homebrew OpenSSL instead of the system version"
diff --git a/Library/Formula/tmux.rb b/Library/Formula/tmux.rb
index 46446ed..1035729 100644
--- a/Library/Formula/tmux.rb
+++ b/Library/Formula/tmux.rb
@@ -33,6 +33,11 @@ class Tmux < Formula
   depends_on "pkg-config" => :build
   depends_on "libevent"
 
+  patch do
+    url "https://gist.githubusercontent.com/choppsv1/352421a9f49262ad5c93/raw/5d29be026e11f75a9f570f354149c913c749a699/tmux-2.1-24bit.diff"
+    sha256 "6d92559da2bf7433f505efd785404fe055772c7cd8c49d208d7522b169c09903"
+  end
+
   def install
     system "sh", "autogen.sh" if build.head?
 
