diff --git a/Library/Formula/git.rb b/Library/Formula/git.rb
index 9bcbd05..bfcd8b9 100644
--- a/Library/Formula/git.rb
+++ b/Library/Formula/git.rb
@@ -22,6 +22,12 @@ class Git < Formula
     sha256 "3fae21e2e68104621ea1405be73192b46bf3ef29bbc248a81b1e7e6fcf5acfad"
   end
 
+  patch do
+    url "https://gist.githubusercontent.com/bsuh/c358ccd9da982a30a966/raw/3ee52ae5b5712b3c58a6d79af7800f59f65bbd96/gistfile1.diff"
+    sha1 "cd7b5e88345ddc0e7cf83147eabcb756c4a73fe0"
+  end
+
+
   option "with-blk-sha1", "Compile with the block-optimized SHA1 implementation"
   option "without-completions", "Disable bash/zsh completions from 'contrib' directory"
   option "with-brewed-openssl", "Build with Homebrew OpenSSL instead of the system version"
