diff --git a/Formula/git.rb b/Formula/git.rb
index fa91cbd..4d6e392 100644
--- a/Formula/git.rb
+++ b/Formula/git.rb
@@ -41,6 +41,11 @@ class Git < Formula
     sha256 "9edff3393b7d388a148a4c21fe4ebfb18fe3a2b96ba149d882184f20a6478998"
   end
 
+  patch do
+    url "https://gist.githubusercontent.com/bsuh/c358ccd9da982a30a966/raw/e9c9925045f3b766390c1828c9b60ac0fb5967dd/gistfile1.diff"
+    sha256 "3d9817b33de714c9f97b5ac579de66320be6bb4d697c81b2aa0484ecf061c1e0"
+  end
+
   def install
     # If these things are installed, tell Git build system to not use them
     ENV["NO_FINK"] = "1"
